python/send_message/main.py

```diff
@@ -0,0 +1,36 @@
+from .functions.send_sms_twilio import send_sms_twilio
+from .functions.send_email_mailgun import send_email_mailgun
+from .functions.send_tweet import send_tweet
+from .functions.send_message_discord_webhook import send_message_discord_webhook
+
+
+def main(req, res):
+    """Main Function"""
+
+    global result
+
+    try:
+        payload = req.payload
+
+        payload_type = payload["type"]
+        message = payload["message"]
+
+        if Type == "SMS":
```

Comment:
Since the variable has been changed to `payload_type` the if statements need to be updated to use the new variable as well.

Reply:
Ok fixed it